### PR TITLE
[TAS-1633] 💄 Update tipping select UI

### PR DESCRIPTION
**Pull request by AuroraHuang22**
截圖 2024-05-23 上午11 41 04
截圖 2024-05-23 上午11 41 08

### CODE REVIEW

File: src/components/NFTBook/TippingDialog.vue [hunk 1]
```diff
@@ -50,16 +58,54 @@
...
+                'rounded-[50%]',
+                'bg-like-green',
+                'border-[1px]',
+                'border-white',
+                'absolute',
+                'top-[-10px]',
+                'left-[-10px]',
+              ]"
+            >
+              <IconCheckMini class="text-white" />
```
Comment (nwingt) [inline src/components/NFTBook/TippingDialog.vue:78]: How about making the SVG absolute and removing the parent div

File: src/components/NFTBook/TippingDialog.vue [hunk 2]
```diff
@@ -50,16 +58,54 @@
...
+              'bg-like-green',
+              'border-[1px]',
+              'border-white',
+              'absolute',
+              'top-[-10px]',
+              'left-[-10px]',
+            ]"
+          >
+            <IconCheckMini class="text-white" />
+          </div>
```
Comment (nwingt) [inline src/components/NFTBook/TippingDialog.vue:108]: Seems can be refactorized

Comment (AuroraHuang22) [reply src/components/NFTBook/TippingDialog.vue]: [♻️ Refactor check icon](https://github.com/likecoin/liker-land/pull/1621/commits/1bf6a7014400f7febed25c82b71fa09ce31bc088)

File: src/components/NFTBook/TippingDialog.vue (hunk 1 above)
Comment (AuroraHuang22) [reply src/components/NFTBook/TippingDialog.vue]: The parent div contains a circular background

**Pull request merged**